packages/jsii-dotnet-generator/src/Amazon.JSII.Generator/Interface/InterfaceDefaultPropertyGenerator.cs

```diff
@@ -13,9 +13,9 @@ public class InterfaceDefaultPropertyGenerator : PropertyGeneratorBase<Interface
         public InterfaceDefaultPropertyGenerator(InterfaceType type, Property property, ISymbolMap symbols, INamespaceSet namespaces)
             : base(type, property, symbols, namespaces)
         {
-            if (property.IsAbstract == true)
+            if (property.IsAbstract == false)
```

Comment:
This should be `if (property.IsAbstract != true)`. `IsAbstract` is a nullable property, so changing from `... == true` to `... == false` also changes the meaning of `null` from "is not abstract" to "is abstract".

Reply:
✔️